templates/activity/parts/activity-consumption.hbs

```diff
@@ -49,9 +44,13 @@
                 </button>
             </div>
         </div>
-        {{else}}
-        <div class="empty">{{ localize "DND5E.None" }}</div>
         {{/each}}
+        <button type="button" class="unbutton control-button add-button" data-action="addConsumption"
+                data-tooltip="DND5E.CONSUMPTION.Action.Create"
+                aria-label="{{ localize "DND5E.CONSUMPTION.Action.Create" }}">
```

Comment:
Since the buttons have the labels inside them, no need for tooltips or `aria-label`.
```suggestion
        <button type="button" class="unbutton control-button add-button" data-action="addConsumption">
```